<launch>
        <group ns="driver">
                <node name="power_node" pkg="seabot_power_driver" type="power_node" output="screen" ns="power" respawn="true" respawn_delay="5">
                        <param name="frequency" value="0.2" />
                </node>

                <node name="piston_node" pkg="seabot_piston_driver" type="piston_node" output="screen" ns="piston" respawn="true" respawn_delay="5">
                        <param name="frequency" value="2.0" />
                </node>

                <node name="pressure_sensor_ext" pkg="pressure_89bsd_driver" type="pressure_89bsd_node" output="screen" respawn="true" respawn_delay="5">
                        <param name="frequency" value="5.0" />
                </node>

                <node name="pressure_sensor_int" pkg="pressure_bme280_driver" type="bme280_node" output="screen" respawn="true" respawn_delay="5">
                        <param name="frequency" value="5.0" />
                </node>

<!--                <node name="thruster_node" pkg="seabot_thruster_driver" type="thruster_node" output="screen" ns="thruster">
                  <param name="frequency" value="15.0" />
                </node>

                <node name="gnss_node" pkg="nmea_navsat_driver" type="nmea_serial_driver.py" output="screen" respawn="true" respawn_delay="5">
                  <param name="port" value="/dev/gps" />
                  <param name="baud" value="115200" />
                  <param name="useRMC" value="True" />
                </node>-->
        </group>
</launch>
